src/components/chip/chip.tsx

```diff
@@ -0,0 +1,278 @@
+import {
+    makeEnterClickable,
+    removeEnterClickable,
+} from '../../util/make-enter-clickable';
+import translate from '../../global/translations';
+import {
+    BACKSPACE,
+    BACKSPACE_KEY_CODE,
+    DELETE,
+    DELETE_KEY_CODE,
+} from '../../util/keycodes';
+import { Chip as OldChipInterface } from '../chip-set/chip.types';
+
+interface ChipInterface extends Omit<OldChipInterface, 'id'> {}
+
+/**
+ * Chips and buttons are both interactive elements in UI design,
+ * but they serve different purposes and are used in different contexts.
+ *
+ * :::warning
+ * Do not use the chip component carelessly, as an alternative for
+ * [`limel-button`](#/component/limel-button/) in the UI design!
+ *
+ * **Buttons:**
+ * Buttons are used to trigger actions. They are typically used to
+ * submit forms, open dialogs, initiate a process, or perform any action
+ * that changes the state of the application.
+ * Buttons' labels usually contain action words, in other words, the labels is
+ * a _verb in imperative mood_ such as "Submit" or "Delete".
+ * Buttons are placed in areas where it's clear they will initiate
+ * an action when clicked.
+ *
+ * **Chips:**
+ * Chips however are elements which may look like buttons, but they are
+ * representing choices, filters, or tags, in a small block
+ * or clearly bundled into a group. Chips are rarely used alone in the
+ * user interface.
+ * They are often used in a so called "chip-set", or placed together in
+ * a section of the UI, where the user can expect more than one chip to be present.
+ *
+ * For example, a chip may represent a filter in a filter bar, or a tag in a tag list,
+ * or an item in a shopping list.
+ * Clicking a chip can also trigger an action, for example toggling a filter ON or OFF,
+ * or opening a page with all posts tagged with the tag represented by the chip,
+ * or navigating to a page with more information about the item in the shopping list.
+ * :::
+ *
+ * @private
+ * @exampleComponent limel-example-chip-button
+ * @exampleComponent limel-example-chip-link
+ * @exampleComponent limel-example-chip-icon-colors
+ * @exampleComponent limel-example-chip-badge
+ * @exampleComponent limel-example-chip-filter
+ * @exampleComponent limel-example-chip-removable
+ * @exampleComponent limel-example-chip-aria-role
+ */
+@Component({
+    tag: 'limel-chip',
+    shadow: true,
+    styleUrl: 'chip.scss',
+})
+export class Chip implements ChipInterface {
+    /**
+     * Defines the language for translations.
+     * Will translate the translatable strings on the components.
+     */
+    @Prop({ reflect: true })
+    public language: Languages = 'en';
+
+    /**
+     * Label displayed on the chip
+     */
+    @Prop({ reflect: true })
+    public text: string;
+
+    /**
+     * Icon of the chip.
+     */
+    @Prop()
+    public icon: string | Icon;
+
+    /**
+     * If supplied, the chip will become a clickable link.
+     */
+    @Prop()
+    public link?: Omit<Link, 'text'>;
```

Comment:
Hmm… I just realised we probably want to adjust this typing just a tad. Right now, I'm pretty sure you can't supply a `Link`, and I think it would be quite nice if you could (just that if you do, we'll ignore the `text` property on it).

I really want to get this merged though, so might look at tweaking that in a separate PR.

Reply:
Ah! OK, `text` is already optional on `Link`, so it's already possible to create a `Link` without a `text` property, and then pass it to `limel-chip`.

I wonder if it's worth omitting the `text` property then… 🤔

I guess it makes it _obvious_ that we won't use the `text` value, if nothing else 😄